monai/apps/deepgrow/interaction.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import importlib
+from typing import Dict
+
+import torch
+
+from monai.engines.utils import CommonKeys
+from monai.engines.workflow import Engine, Events
+from monai.transforms import Compose
+
+
+class Interaction:
+    """
+    Deepgrow Training/Evaluation iteration method with interactions (simulation of clicks) support for image and label.
```

Comment:
Please highlight in the doc-string that this class is an ignite handler.

Reply:
fixed